calc/async_calc_graph.go

```diff
@@ -99,7 +115,18 @@ func (acg *AsyncCalcGraph) loop() {
 			case []api.Update:
 				// Update; send it to the dispatcher.
 				log.Debug("Pulled []KVPair off channel")
+				updStartTime := time.Now()
 				acg.Dispatcher.OnUpdates(update)
+				updEndTime := time.Now()
+				if updEndTime.After(updStartTime) {
```

Comment:
Worth commenting that this is to do with protecting against time jumping backwards.

What if it jumps forwards, though?